.github/actions/set-github-https-auth/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Configure git to use bot credentials'
+description: 'Configure git to use bot credentials for private repositories using git config'
+inputs:
+  bot-username:
+    description: 'The username of the bot account'
+    required: true
+  bot-token:
+    description: 'The token of the bot account'
+    required: true
```

Comment:
don't use `bot-` prefix, they can be whatever credentials

token may be `pat` which is a common word in github credentials